Sources/VaporToolbox/New/TemplateScaffolder.swift

```diff
@@ -124,27 +124,35 @@ struct TemplateScaffolder {
                 }
             }
         }
+        
+        let destinationFileName: String
+        if let dynamicName = file.dynamicName {
+            destinationFileName = try MustacheRenderer().render(template: dynamicName, data: context)
+        } else {
+            destinationFileName = file.name
+        }
+        let destinationPath = destination.appendingPathComponents(destinationFileName)
 
         switch file.type {
         case .file(let dynamic):
             self.console.output("+ " + file.name.consoleText())
             if dynamic {
                 let template = try String(contentsOf: source.appendingPathComponents(file.name).asFileURL, encoding: .utf8)
                 try MustacheRenderer().render(template: template, data: context)
```

Comment:
We're creating the `MustacheRenderer` in and on line 130 - we should pull this out so it's only initialised once

Reply:
@0xTim Thanks for the feedback.

I have commited a change so that only one `MustacheRenderer` is created.